rusk-wallet/Cargo.toml

```diff
@@ -23,7 +23,7 @@ bs58 = "0.4"
 rand = "0.8"
 aes = "0.7"
 
-dusk-wallet-core = "0.10.0-rc"
+dusk-wallet-core = { git = "https://github.com/dusk-network/wallet-core", branch = "stake-change-46" }
```

Comment:
Should this really point to a git branch? 🧐

Reply:
No it shouldn't. It currently does because wallet wallet core hasn't yet landed version `0.11.0`